.gitignore

```diff
@@ -1,5 +1,6 @@
 .*
 
+!.idea/
```

Comment:
Could you add a comment explaining this, e.g. `# Includes some IntelliJ-specific configuration - unnecessary items are ignored in .idea/.gitignore`